### DISCUSSION TITLE
"central word" và "central target word"

### DISCUSSION THREAD

**Discussion by trgiangdo** [d2l-ai/d2l-vi#2550] [Terminology]
Trong #2499 anh @cuongvng có đề xuất đồng nhất **central word** và **central target word**, hoặc dịch **central target word** là **từ đích trung tâm**.

> chỗ này mình thấy nhiều chỗ chỉ để là central word, có thể bỏ target đi ko nhỉ? trước giờ vẫn đang dịch là đích.

Hiện em thấy trong glossary có dịch **target data / distribution** là **dữ liệu / phân phối mục tiêu** nên em dịch theo thành **từ mục tiêu trung tâm**.

cc: @minhduc0711 @cuongvng @lkhphuc @davidnvq @rootonchair

**Comment by quangvnai** [comment]
Anh thấy cái từ **dữ liệu / phân phối mục tiêu** nghe cũng tạm được. Nhưng bên NLP, thì họ dùng từ **đích** phổ biến hơn. 

Cụm `source A` - `target A` chẳng hạn:
* ngôn ngữ nguồn - ngôn ngữ đích.
* từ nguồn - từ đích.
* ...

Đừng hiểu lẫn **central word** và **central target word** là một khi gặp một vài từ như vậy. Vì trong ngữ cảnh đấy thì viết ngắn gọn không gây cho người đọc hiểu nhầm sang từ khác. Nếu **central source word** thì dịch sao?

#### References:
* [HUI - Ứng dụng Xử lý Ngôn ngữ Tự nhiên trong Dịch máy](http://www.hui.edu.vn/Resource/Upload/file/NCKH/KHCN_tuyentapBOCONGTHUONG/15.%20ungdungxulyngonngutunhien_NCHieu.pdf).

**Reply by trgiangdo** [reply]
Nếu vậy em nghĩ nên thêm `target word` vào glossary.

**Reply by lkhphuc** [reply]
đúng rồi, cái `phân phối mục tiêu` mình thống nhất cách xài cho các phần liên quan tới transfer learning.
Các ngư cảnh khác không nhất thiết phải dịch là `mục tiêu` hết mà linh động `mục tiêu / đích` tuỳ trường hợp.

**Reply by lkhphuc** [reply]
Mình prefer cách dịch `central target word` là `từ tâm đích` của bác @LongNguyen1984 .
Mình có thể vote và thêm vào glossary hoặc chỉ đơn giản là sửa lại đồng loạt khi revise cũng được, vì cụm này chỉ đặc biệt xài trong ngữ cảnh word embedding thôi.

**Reply by trgiangdo** [reply]
Ok vậy em sẽ sửa các PR hiện tại của em theo cách dịch này. Còn đâu trong lúc revise anh sửa lại 1 loạt nhé.

**Comment by lkhphuc** [comment]
Thắc mắc của @cuongvng là có bỏ `target` đi được không thì mình thấy là không.
Central / Context là chỉ vị trí của từ, còn `target` hay `source` là chỉ model `CBOW` hoặc `skip-gram`.
Họ xài central khi chỉ cần nhấn mạnh vị trí của từ, thêm `target` là để nhấn mạnh vị trí và vai trò của từ.

**Comment by rootonchair** [comment]
Mình mở PR để add từ này vào glossary nhỉ mọi người? + Rà soát lại chương để sửa lại cách dịch cho từ  này

**Reply by quangvnai** [reply]
@rootonchair  đã mở PR rồi nhé. Có điều `từ tâm` đọc nhanh giống như cụm từ "from the heart" trong đạo Phật quá :-?

**Reply by rootonchair** [reply]
Em vừa có cùng suy nghĩ đó ạ 😆 
@lkhphuc anh thấy thêm thành `trung tâm` thì sao ạ?

**Reply by lkhphuc** [reply]
Vậy chắc mình cứ xài `từ đích trung tâm` nhỉ, đúng thứ tự đảo ngược của `central target word` hơn là `tâm đích`.
